python/packages/ai/teams/ai/augmentations/sequence_augmentation.py

```diff
@@ -179,5 +185,16 @@ async def create_plan_from_response(
             Plan: The created plan
         """
         if response.message and response.message.content:
-            return response.message.content
+            plan = response.message.content
+            plan.commands = [
+                (
+                    PredictedSayCommand(
+                        response=Message(role="assistant", content=command.response.content)
```

Comment:
this is the JS bug I fixed this morning, you will need to add `response.message.context` to this message to ensure citations work with sequence augmentation.